srcpkgs/goatcounter/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'goatcounter'
+pkgname=goatcounter
+version=1.2.0
+revision=1
+wrksrc=${pkgname}-${version}
+build_style=go
+go_import_path=zgo.at/goatcounter
+short_desc="Easy web analytics. No tracking of personal data"
+maintainer="Anjandev Momi <anjan@momi.ca>"
+license="EUPL-1.2"
+homepage="https://github.com/zgoat/goatcounter"
+distfiles="https://github.com/zgoat/goatcounter/archive/v${version}.tar.gz"
+checksum=505f8c5848bb368e18c0a041fcbeb87f540813a7455d9a67cdce5a5f25d799e7
+
+do_build() {
+	go build ./cmd/goatcounter
```

Comment:
Should be possible to be handled by setting `go_package="${go_import_path}/cmd/goatcounter"` Also should allow to remove `do_install`